### PR TITLE
5907 - datagrid tooltip flashes continuously

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will fix a bug in datagrid where tooltip flashes continuously on hover.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5907

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/datagrid/test-custom-long-tooltip.html
- Hover over cell in the `Exception` column

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by tjamesallen15** [comment]
Will fix the E2E after this is reviewed.

File: src/components/datagrid/datagrid.js [hunk 1]
```diff
@@ -5590,6 +5590,11 @@ Datagrid.prototype = {
       delay = typeof delay === 'undefined' ? defaultDelay : delay;
       clearTimeout(tooltipTimer);
       setTimeout(() => {
+        // Prevents the tooltip to show on and off
+        if ($('.grid-tooltip:hover').length > 0) {
```
Comment (ericangeles) [inline src/components/datagrid/datagrid.js:5594]: Did you try to use `.grid-tooltip:not(.is-hidden)` class instead of :hover selector?

Comment (tjamesallen15) [reply src/components/datagrid/datagrid.js:5594]: yes. it somehow doesn't work thats why I used the hover selector.

**Review by tmcconechy — CHANGES_REQUESTED** [review]
I think this works pretty good except i would test more.. I see at least one issue going through:

http://localhost:4000/components/datagrid/ (names have tooltip in them)
compare to https://main-enterprise.demo.design.infor.com/components/datagrid/

a) if i move the mouse down the column the tooltips dont show for each cell anymore.
- go to http://localhost:4000/components/datagrid/example-tooltips.html
- hover the arrow on row one and get the tooltip
- hover  the arrow on row two and i no longer get the tooltip
- compare to https://main-enterprise.demo.design.infor.com/components/datagrid/example-tooltips.html
b) same on the example with the big tooltip http://localhost:4000/components/datagrid/test-custom-long-tooltip.html moving down

**Review by tmcconechy — DISMISSED** [review]

**Comment by tmcconechy** [comment]
@tjamesallen15 fix looks good but the build is failing https://github.com/infor-design/enterprise/actions/runs/3287748709/jobs/5417584258#step:8:2754 maybe we need to update this one?

**Comment by tjamesallen15** [comment]
> @tjamesallen15 fix looks good but the build is failing https://github.com/infor-design/enterprise/actions/runs/3287748709/jobs/5417584258#step:8:2754 maybe we need to update this one?

Yes, yes. Will be updating today.

**Review by tmcconechy — DISMISSED** [review]

**Review by tmcconechy — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**